low-level/framework/sspl_ll_d

```diff
@@ -819,7 +819,7 @@ if __name__ == "__main__":
         main(conf_reader, systemd_support)
 
     except (Exception) as err:
-        logger.debug(f"{traceback.print_exc()}")
+        logger.exception(f"{traceback.print_exc()}")
```

Comment:
When you use logger.exception, exception info automatically gets added.
So you can directly do logger.exception("Error while initializing")

Reply:
Done.